bowtie/_cli.py

```diff
@@ -1357,6 +1359,32 @@ async def filter_dialects(
             break
 
 
+@subcommand
+@click.option(
```

Comment:
This is essentially redefining `@DIALECT`.

Reply:
But the `@DIALECT` doesn’t have a default 2020-12 on it I suppose.